sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -884,6 +1005,14 @@ def __repr__(self):
             :1024
         ]
 
+    def to_dict(self):
+        return {
+            "model_id": self.model_id,
+            "accuracy": self.accuracy,
+            "fields": [v.to_dict() for v in self.fields] if self.fields else [],
```

Comment:
`fields` should also return dict here

```suggestion
            "fields": {k: v.to_dict() for k, v in self.fields.items()} if self.fields else {},
```

Reply:
Done